### PR TITLE
README: Fix example

### PR DESCRIPTION
**Pull request by carlmjohnson**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -9,15 +9,13 @@ import (
...
-	"os"
 
 	"github.com/apex/gateway"
-	"github.com/aws/aws-lambda-go"
 )
 
 func main() {
 	http.HandleFunc("/", hello)
-	log.Fatal(gateway.ListenAndServe(":3000", nil))
+	log.Fatal(gateway.ListenAndServe("n/a", nil))
```
Comment (carlmjohnson) [inline Readme.md]: Docs should also be more clear that the address is just there to let you swap with `http.ListenAndServe` but doesn't do anything.

**Comment by carlmjohnson** [comment]
#25 didn't remove os from the imports, so this is still relevant.

File: Readme.md (hunk 1 above)
Comment (tj) [reply Readme.md]: sounds reasonable, but now it won't run locally. Maybe we should just have a comment before the ListenAndServe or something

Comment (carlmjohnson) [reply Readme.md]: In any real code, you need to have to something like 

```go
	listener := http.ListenAndServe
	if isLambda {
		listener = gateway.ListenAndServe
	}
	log.Fatal(listener(port, handler))
```

anyway, so I don't think it matters if the README code technically can't be run locally, as long as it compiles.

**Pull request closed without merging**